src/ui/layout/CreateButton.vue

```diff
@@ -74,23 +73,9 @@ export default {
             this.openmct.menus.showSuperMenu(x, y, this.sortedItems, menuOptions);
         },
         create(key) {
-            // Hack for support.  TODO: rewrite create action.
-            // 1. Get contextual object from navigation
-            // 2. Get legacy type from legacy api
-            // 3. Instantiate create action with type, parent, context
-            // 4. perform action.
-            return this.openmct.objects.get(this.openmct.router.path[0].identifier)
```

Comment:
We didn't lose the context by removing this get right?

Reply:
I might be misunderstanding the issue here, but this get looks redundant to me. 

We were `get` the currently navigated object, which would always just return the domain object itself, without context.

We already have the navigated object, so there was no need to `get` it again. Maybe there's some nuance I'm missing here?